CONTRIBUTORS.md

```diff
@@ -1,3 +1,5 @@
+<!-- Copyright 2018-2022 contributors to the OpenLineage project -->
```

Comment:
This information already is below:

`<!-- Copyright Contributors to the OpenLineage. -->`